mapsims/noise.py

```diff
@@ -292,6 +293,26 @@ def get_noise_spectra(self, tube, ncurve_sky_fraction=1):
             the corresponding sky fraction determined from each
             band's hitmap.
 
+        return_corr : bool
+            If True, returns cross-correlation coeffient instead of
+            cross-correlation power in the third row of the returned arrays.
```

Comment:
can you expand a bit the docstring about `return_corr`? in which case you would use it?

Reply:
Done.